apps/web/app/(app)/environments/[environmentId]/integrations/mattermost/components/ManageIntegration.tsx

```
@@ -0,0 +1,132 @@
+                    key={index}
+                    className="m-2 grid h-16 grid-cols-8 content-center rounded-lg text-slate-700 hover:cursor-pointer hover:bg-slate-100"
+                    onClick={() => {
+                      editIntegration(index);
+                    }}>
+                    <div className="col-span-2 text-center">{data.surveyName}</div>
+                    <div className="col-span-2 text-center">{data.channelName}</div>
+                    <div className="col-span-2 text-center">{data.questions}</div>
+                    <div className="col-span-2 text-center">{timeSince(data.createdAt.toString())}</div>
+                  </div>
+                );
+              })}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)